2017/CG-05.md

```diff
@@ -93,7 +93,7 @@ Please register before the event [here](https://goo.gl/forms/LQVhuRuI22lK8Umw1).
 
 | Dates                    | Location          | Host       |
 |--------------------------|-------------------|------------|
-| 2017-07-17 to 2017-07-19 | San Francisco, CA | Google     |
+| 2017-07-17 to 2017-07-19 | Kirkland,WA       | Google     |
```

Comment:
There was consensus to change this to 18th through 20th July, can someone confirm?